javascript/hand-gesture-recognizer/gestures.js

```diff
@@ -0,0 +1,340 @@
+            1
+          ],
+          [
+            "setWeight",
+            "Thumb",
+            2
+          ]
+        ],
+        "enabled": true
+      },
+  stopGesture:{
+    "name": "stop",
+    "algorithm": "fingerpose",
+    "models": "hands",
+    "confidence": "7",
+    "description": [
+      [
+        "addCurl",
+        "Thumb",
+        "NoCurl",
+        1
+      ],
+      [
+        "addDirection",
+        "Thumb",
+        "DiagonalUpLeft",
+        1
+      ],
+      [
+        "addCurl",
+        "Index",
+        "NoCurl",
+        1
+      ],
+      [
+        "addDirection",
+        "Index",
+        "DiagonalUpLeft",
+        1
+      ],
+      [
+        "addCurl",
+        "Middle",
+        "NoCurl",
+        1
+      ],
+      [
+        "addDirection",
+        "Middle",
+        "VerticalUp",
+        1
+      ],
+      [
+        "addCurl",
+        "Ring",
+        "NoCurl",
+        1
+      ],
+      [
+        "addDirection",
+        "Ring",
+        "VerticalUp",
+        1
+      ],
+      [
+        "addCurl",
+        "Pinky",
+        "NoCurl",
+        1
+      ],
+      [
+        "addDirection",
+        "Pinky",
+        "VerticalUp",
+        1
+      ]
+    ],
+    "enabled": true
+  },
+  okGesture: {
+    "name": "ok",
+    "algorithm": "fingerpose",
+    "models": "hands",
+    "confidence": "7",
+    "description": [
+      [
+        "addCurl",
+        "Thumb",
+        "NoCurl",
+        1
+      ],
+      [
+        "addCurl",
+        "Thumb",
+        "HalfCurl",
+        0.034482758620689655
+      ],
+      [
+        "addDirection",
+        "Thumb",
+        "DiagonalUpLeft",
+        1
+      ],
+      [
+        "addCurl",
+        "Index",
+        "HalfCurl",
+        1
+      ],
+      [
+        "addCurl",
+        "Index",
+        "FullCurl",
+        0.5263157894736842
+      ],
+      [
+        "addCurl",
+        "Index",
+        "NoCurl",
+        0.05263157894736842
+      ],
+      [
+        "addDirection",
+        "Index",
+        "DiagonalUpLeft",
+        1
+      ],
+      [
+        "addCurl",
+        "Middle",
+        "NoCurl",
+        1
+      ],
+      [
+        "addDirection",
+        "Middle",
+        "DiagonalUpLeft",
+        1
+      ],
+      [
+        "addCurl",
+        "Ring",
+        "NoCurl",
+        1
+      ],
+      [
+        "addDirection",
+        "Ring",
+        "DiagonalUpLeft",
+        1
+      ],
+      [
+        "addDirection",
+        "Ring",
+        "VerticalUp",
+        0.034482758620689655
+      ],
+      [
+        "addCurl",
+        "Pinky",
+        "NoCurl",
+        1
+      ],
+      [
+        "addDirection",
+        "Pinky",
+        "VerticalUp",
+        1
+      ],
+      [
+        "addDirection",
+        "Pinky",
+        "DiagonalUpLeft",
+        0.1111111111111111
+      ],
+      [
+        "setWeight",
+        "Index",
+        2
+      ]
+    ],
+    "enabled": true
+  }
+}
```

Comment:
Nitpick: Let's add an empty line at the end of the file.
```suggestion
}

```